src/solver/conv_hip_implicit_gemm_grouped_fwd_xdlops.cpp

```diff
@@ -214,7 +340,17 @@ bool PerformanceConfigHipImplicitGemmGroupFwdXdlops::SetNextValue(const ProblemD
 {
     if(valid_kernels.empty())
     {
-        HeuristicInit(problem);
+        switch(problem.GetInDataType())
+        {
+        case miopenHalf: Init<ck::half_t>(problem); break;
+        case miopenFloat: Init<float>(problem); break;
+        case miopenInt8: Init<int8_t>(problem); break;
+        case miopenInt32:
+        case miopenBFloat16:
+        case miopenFloat8:
+        case miopenBFloat8:
+        case miopenDouble: break;
```

Comment:
Shouldn't this throw if we haven't initialized with anything?

Reply:
The assert on line 354 should throw an error.